pkgs/top-level/all-packages.nix

```diff
@@ -2824,6 +2824,8 @@ in
 
   fltrdr = callPackage ../tools/misc/fltrdr { stdenv = gcc8Stdenv; };
 
+  fluentbit = callPackage ../tools/misc/fluent-bit { };
```

Comment:
Package name should be consisent with folder name, derivation name, and preferably, executable name. `fluent-bit` seems to be the name chosen by upstream.